src/main/java/bookstore/service/category/CategoryService.java

```diff
@@ -0,0 +1,18 @@
+package bookstore.service.category;
+
+import bookstore.dto.category.CategoryDto;
+import bookstore.dto.category.CreateCategoryRequestDto;
+import java.util.List;
+import org.springframework.data.domain.Pageable;
+
+public interface CategoryService {
+    List<CategoryDto> findAll(Pageable pageable);
+
+    CategoryDto getById(Long id);
```

Comment:
It's possible to get null, we have a wrapper that deals with nulls

Reply:
Optional used on repository level, so you can leave it as it is